src/defid.cpp

```diff
@@ -64,6 +66,9 @@ static bool AppInit(int argc, char* argv[])
     bool fRet = false;
 
     util::ThreadRename("init");
+    init_runtime();
```

Comment:
Remove initialization of EVM for now. We'll do it in separate branch.

Reply:
This is the gRPC server runtime